### PR TITLE
Remove in-app doc links to powerofus

### PR DESCRIPTION
**Pull request by AndersonTarren**
# Critical Changes

# Changes
Updates in-app documentation links

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by AndersonTarren** [comment]
[W-11721647](https://gus.lightning.force.com/a07EE000015w2ADYAY)

File: src/pages/VolunteersAbout.page [hunk 1]
```diff
@@ -55,7 +55,7 @@
             <div class="slds-col slds-size--1-of-6"  id="colStep3">
                 <div class="slds-text-align--center slds-p-bottom--small">{!$Label.labelVolunteersHelpStep3}</div>
                 <figure class="slds-image">
-                    <a href="https://powerofus.force.com/V4S_Group" class="slds-align--absolute-center">
+                    <a href="https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHiiSAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC" class="slds-align--absolute-center">
```
Comment (jjbennett) [inline src/pages/VolunteersAbout.page:58]: Should we also update the image? We could maybe just anchor the Trailhead image?
Screen Shot 2022-09-21 at 1 09 55 PM

Comment (AndersonTarren) [reply src/pages/VolunteersAbout.page:58]: Yea, we have to do something about this image.  I was going to demo this on the docs call this afternoon.

Comment (jjbennett) [reply src/pages/VolunteersAbout.page:58]: ok cool, links work. I will test again to make sure the image looks a'ok!

**Comment by AndersonTarren** [comment]
@fanwar-salesforce this one should be ready.  I made adjustments to the installer and about pages.

**Review by fanwar-salesforce — APPROVED** [review]
Checked that there are no Power of US URLs anymore. Looks good!

**Pull request merged**